File: lib-google-proximity/src/main/java/com/ndipatri/iot/googleproximity/utils/BluetoothHelper.java
package com.ndipatri.iot.googleproximity.utils;


import android.bluetooth.BluetoothAdapter;

public class BluetoothHelper {

    public boolean isBluetoothSupported() {
        return BluetoothAdapter.getDefaultAdapter() != null;
    }

    public boolean isBluetoothEnabled() {
        return isBluetoothSupported() && BluetoothAdapter.getDefaultAdapter().isEnabled();
    }
}

File: lib-google-proximity/src/main/java/com/ndipatri/iot/googleproximity/utils/BluetoothHelper.kt
package com.ndipatri.iot.googleproximity.utils


import android.bluetooth.BluetoothAdapter

class BluetoothHelper {

    val isBluetoothSupported: Boolean
        get() = BluetoothAdapter.getDefaultAdapter() != null

    val isBluetoothEnabled: Boolean
        get() = isBluetoothSupported && BluetoothAdapter.getDefaultAdapter().isEnabled
}
